beightone/search-result
# Feature/filter selected mobile

#### What problem is this solving?

This feature will include the possibility to show the selected filters on mobile, allowing the user to clear individual filters using this section. A clear-all-filters button will also be added on top os this section, in a similar fashion to the desktop version.

#### How to test it?

Access the [workspace](https://testetestesearch--bricobravo.myvtex.com/test-category) <use the mobile version>
Select some filters and click the apply filters button
Open the filter drawer again and verify the section with the selected filters and clear all filters button

[Workspace](https://testetestesearch--bricobravo.myvtex.com/test-category)

#### Screenshots or example usage:

Obs.: The selected filters checkbox and the clear all filters button have been stylized for the client in this screenshot (this is not the default styling)

![filterSelectedMobile](https://user-images.githubusercontent.com/4131073/185654433-cb033906-6dd9-46c1-a95b-18dc892b6797.png)
